api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -581,6 +581,8 @@ message Features {
   // NOTE: this flag is used to signal that Access Monitoring is *enabled* on a cluster.
   // *Access* to the feature is gated on the `AccessMonitoring` entitlement.
   bool AccessMonitoringConfigured = 36;
+  // cloud_anonymization_key is a hash of the Salesforce ID used to anonymize usage events
+  bytes cloud_anonymization_key = 37 [json_name = "cloudAnonymizationKey"];
```

Comment:
```suggestion
  bytes CloudAnonymizationKey = 37 [ (gogoproto.jsontag) = "cloud_anonymization_key"];
```

Can we be consistent with the other fields in this message? (assuming we can change the JSON key without breaking anything)